pages/jobs.tsx

```diff
@@ -0,0 +1,71 @@
+// import Link from 'next/link'
+import { useState, useEffect } from 'react'
+import { GetStaticProps } from 'next'
+import { siteConfig } from '@/const/meta'
+import Content from 'components/Layout/Content'
+import { Section, Title, SubTitle, TitleSmall, LinkContainer, LoadingText } from 'const/styles/pages/content'
+import { ExternalLink } from '@/const/styles/global'
+import SVG from 'react-inlinesvg'
+
+async function getJobs() {
+  const { api, deptID } = siteConfig.greenhouse
+  const getJobs = await fetch(api)
+  const { jobs } = await getJobs.json()
+  const jobsData = []
+  await jobs.map((job) => {
+    const dept = job.departments[0]
+    if (dept) {
+      dept.id === deptID ? jobsData.push(job) : null
+    }
+  })
+
+  return jobsData
+}
+
+export default function Jobs({ jobsData }) {
+  const discordURL = siteConfig.social.discord.url
+  const [jobs, setJobs] = useState(jobsData)
+  const [isLoading, setLoading] = useState(false)
+
+  // Fetch the latest jobs to possibly override static rendered (getStaticProps) SSR jobs
+  useEffect(() => {
+    setLoading(true)
+    getJobs()
+      .then((data) => {
+        setJobs(data)
+        setLoading(false)
+      })
+  }, [])
+
+  return (
+    <Content>
+      <Section>
+        <Title>Want to build the future of decentralized trading?</Title>
+        <SubTitle>We are an ambitious, fast growing and international team working at the forefront of DeFi. We believe that we can make markets both more efficient and fair, by building the ultimate batch auction settlement layer across EVM compatible blockchains.</SubTitle>
+      </Section>
+
+      <TitleSmall>Who we are looking for:</TitleSmall>
+
+      {isLoading && <LoadingText>Loading jobs...</LoadingText>}
+      {!isLoading && jobs.length > 0 && jobs.map(({ title, absolute_url, location, internal_job_id }) =>
+        <LinkContainer key={internal_job_id} href={absolute_url} target="_blank" rel="noopener nofollow noreferrer">
+          <b>{title}</b>
+          <i>{location.name}</i>
+          <SVG src="images/icons/arrowRight.svg" />
```

Comment:
Why did you use inline SVG? Isn't it better to use a cacheable SVG, specially because this is used for each job?

Reply:
Agreed, in general better to have your image assets cached.

In this case I'm using different colors for the SVG arrow. You can't chance SVG (path) colors unless the SVG is inlined. 

It's true that you could simply have 2 SVG's for each color and embed each image and have that cached individually.

At the same it's nice being able to have control on the SVG colors programmatically leveraging the `theme` colors instead.

The inline SVG's size comes out at ~221 bytes. Which in the context of this page could be regarded a negligible size, favoring the flexibility of using theme colors.

One way to further optimize the use of inline SVG's could be to use a SVG sprite. I might tackle that in this PR already to test it out.

Reply:
nice! thanks for looking into it and taking the time to explain and reason about it